components/dashboard/src/teams/sso/OIDCClientConfigModal.tsx

```diff
@@ -36,11 +38,14 @@ export const OIDCClientConfigModal: FC<Props> = ({ clientConfig, onClose }) => {
 
         try {
             await save(ssoConfig);
+            if (isNew) {
+                toast("Your SSO configuration was created. You may now verify, then activate it");
```

Comment:
**suggestion**: Leaving this here for future improvements.

| BEFORE | AFTER |
|-|-|
| <img width="788" alt="Welcome" src="https://github.com/gitpod-io/gitpod/assets/120486/7e175e41-7cca-4a25-b18c-2b8c19361214"> | <img width="788" alt="Welcome-3" src="https://github.com/gitpod-io/gitpod/assets/120486/44534bf7-19f0-4df0-b911-36b4707f9501"> |
| <img width="788" alt="Welcome-1" src="https://github.com/gitpod-io/gitpod/assets/120486/2c303133-68b2-47ec-be99-e1cd9e2380d5"> | <img width="788" alt="Welcome-2" src="https://github.com/gitpod-io/gitpod/assets/120486/e00a194e-ba22-4b83-9000-c0960fcda21f"> |